libp2p/stream_muxer/mplex/mplex_stream.py

```diff
@@ -58,19 +58,41 @@ def is_initiator(self) -> bool:
         done, pending = await asyncio.wait(  # type: ignore
             [
                 self.event_reset.wait(),
-                self.event_remote_closed.wait(),
                 self.incoming_data.get(),
+                self.event_remote_closed.wait(),
```

Comment:
i thought the matter here wasn't important -- is it?

Reply:
Just checked the source code and you're right it is not.